modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -46,7 +46,7 @@ It checks if the user is already logged in on Bonita
 ====
 
 Bonita `username` should match the `prefered_username` or one claims returned by the OIDC provider in the JWT token in the response.
- If some users need to be able to log in without having an account on the IdP, you can authorize it by activating an option in the file `authenticationManager-config.properties` (see 2. below). Users will then be able to log in using the portal login page (`/login.jsp`) provided they have a Bonita account and their password is different from their username. +
+ If some users need to be able to log in without having an account on the IdP, you can authorize it by activating an option in the file `authenticationManager-config.properties` (see 2. below). Users will then be able to log in using the Bonita login page (`/login.jsp`) provided they have a Bonita account and their password is different from their username. +
```

Comment:
provided they have a Bonita account and their password is different from their username. => with the provided Bonita username and a password that is different from it.

Reply:
Um, the username is not provided, I think provided in this phrase means "if"

Reply:
I'm guessing I can close this one